examples/plotting/hmi_LOSmag.py

```diff
@@ -0,0 +1,66 @@
+"""
+====================
+HMI plotting example
+====================
+In this example it is going to be demonstrated how to download and plot data 
+from HMI. For this example we will use the LOS_magnetic_field but any other HMI
+product can be similarly be accessed just by changing the Physical Observables
+(PHYSOBS) keyword on the search. 
+
+Useful links:
+The list with HMI keys for PHYSOBS_.
```

Comment:
This appears to be breaking the doc build on circle.

I would suggest just using the inline link format:
```
`PHYSOBS <https://sdac.virtualsolar.org/cgi/show_details?keyword=PHYSOBS>`__
```

same goes for all the links.